### PR TITLE
Petition finder: add 'domain filter'

### PR DESCRIPTION
**Pull request by lumannnn**

### CODE REVIEW

File: app/node_modules/petition/finder.js [hunk 1]
```diff
@@ -64,6 +82,35 @@ const ItemsTemplate = ({ item, router }) => {
   return null;
 };
 
+const PetitionQueryStateContainer = (location, router, pathname) => {
+  /**
+   * Extend the 'PetitionQueryStateContainer'.
```
Comment (lusbuab) [inline app/node_modules/petition/finder.js]: QueryStateContainer

File: app/app.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ import { translate } from 'config/strings';
 import authenticated from 'auth';
 import storage from 'helpers/storage';
 import PetitionAPI from 'petition/api';
+import { CITY_PORTAL_PARAM, CITY_PORTAL_CLEAR_FILTER } from 'petition/api';
+import { retreiveCityPortalId } from 'petition/api';
```
Comment (lusbuab) [inline app/app.js]: typo: retr_ie_ve

Comment (lusbuab) [inline app/app.js]: well, I guess it's my fault from a few months ago 😔

Comment (lumannnn) [inline app/app.js]: should already be fixed. i've pushed an additional commit. but github seems to be slow right now and does not show the latest update

**Pull request merged**